scripts/run-zombie-moonbeam.sh

```diff
@@ -0,0 +1,24 @@
+mkdir -p target/release/wbuild/moonbase-runtime/
+mkdir -p test/tmp
+
+# When we remove this step, re-add "runScripts": ["download-polkadot.sh"]," to moonwall config
+cd test/tmp
+wget https://opslayer-dev-artifacts.s3.us-east-2.amazonaws.com/bins/moonbeam/polkadot/1.3.0/polkadot
```

Comment:
This is just the relay binary, I removed the script since I only used it locally. (The github jobs use the same binary, the Ops team will probably add new binaries later and we can update the CI, but for now it is fine)